components/rating/index.css

```diff
@@ -43,8 +43,10 @@
 	--spectrum-rating-icon-count: var(--spectrum-rating-icon-count);
 }
 
+/* stylelint-disable-next-line no-duplicate-selectors */
 .spectrum-Rating {
-	&.is-focused {
```

Comment:
In the `.spectrum-Rating--emphasized` block, there's a `&.is-focused`. Do we need to update that too, to `.is-keyboardFocused`? I feel like it might have something to do with this? Where if I have `isEmphasize: true`, the keyboard focus styles remain the dark gray.

https://github.com/user-attachments/assets/ad4f94c0-c52c-4e81-a3f3-6702d61d1b35

Reply:
There might be more color conflicts that are definitely a separate PR actually. With `isEmphasized: true` again, if I hover juuuust right, I can see both the gray unselected icon, or the blue unselected icon. I have a feeling it's somewhere near this code? I think it's just supposed to be gray the whole time, not sure at all though hahah

https://github.com/adobe/spectrum-css/blob/c328acd63a780de14530d12f213553b1d70eb14e/components/rating/index.css#L213

Reply:
_Oof_ haha — I swapped the `.is-focused` to `.is-keyboardFocused`, thanks for catching that!

<img width="180" alt="Screenshot 2024-10-10 at 3 15 38 PM" src="https://github.com/user-attachments/assets/a5d38a0f-2224-4ee8-89d3-c05da2ea13a6">

The blue unselected border color is coming from `fill: currentColor` here:

<img width="251" alt="image" src="https://github.com/user-attachments/assets/a096bae9-e71f-4bfb-8c02-ef2166656617">

I suppose we can try and override it, but the icons are getting set based on the `color` set in the component. https://www.w3.org/TR/css-color-3/#currentcolor